test/test_webhook_grafana.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+# See LICENSE in the project root for license information.
+
+from falcon import HTTPBadRequest
+
+
+def test_parse_valid_body():
+    from iris.webhooks.grafana import grafana
+    grafana_webhook = grafana()
+
+    fake_post = {
+        "evalMatches": [{
+            "value": 100,
+            "metric": "High value",
+            "tags": "",
+        }, {
+            "value": 200,
+            "metric": "Higher Value",
+            "tags": "",
+        }],
+        "imageUrl": "http://grafana.org/assets/img/blog/mixed_styles.png",
+        "message": "Someone is testing the alert notification within grafana.",
+        "ruleId": 0,
+        "ruleName": "Test notification",
+        "ruleUrl": "https://grafana.org/",
+        "state": "alerting",
+        "title": "[Alerting] Test notification"
+    }
+    grafana_webhook.validate_post(fake_post)
+
+
+def test_parse_invalid_body():
+    from iris.webhooks.grafana import grafana
+    grafana_webhook = grafana()
+
+    fake_post = {
+        "evalMatches": [{
+            "value": 100,
+            "metric": "High value",
+            "tags": "",
+        }, {
+            "value": 200,
+            "metric": "Higher Value",
+            "tags": "",
+        }],
+        "imageUrl": "http://grafana.org/assets/img/blog/mixed_styles.png",
+        "message": "Someone is testing the alert notification within grafana.",
+        "ruleId": 0,
+        "ruleName": "Test notification",
+        "ruleUrl": "https://grafana.org/",
+        "state": "alerting",
+        "title": "[Alerting] Test notification"
+    }
+    try:
```

Comment:
Why not use something like the following to ensure the exception does get raised? With the current method of catching It we don't know for sure if it was raised or not.

```
with pytest.raises(HTTPBadRequest):
    grafana_webhook.validate_post(fake_post)
```

Reply:
https://github.com/linkedin/iris/pull/486